index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+  <body>
+    <header class="container">
+      <nav class="top-bar">
+        <div class="status">
+          <div id="time-container">
+            <img
+              id="time-icon"
+              src="images/Time Style.png"
+              alt="not available!"
+              class="time"
+            />
+          </div>
+          <div id="status-container">
+            <img
+              id="status-icon"
+              src="images/Group 3.png"
+              alt="not available!"
+              class="internet-wifi-charge"
+            />
+          </div>
+        </div>
+        <div class="header">
+          <div class="text-welcome">
+            <p class="welcome">
+              <a href="#">Welcome</a>
+            </p>
+          </div>
+          <nav class="humberger-icon-container">
+            <img
+              id="humberger-icon"
+              src="images/Enabled.png"
+              alt="not available"
+            />
+          </nav>
+        </div>
```

Comment:
- [ ] **[OPTONAL]** We could put "Welcome" and the hamburger button inside the `nav` tag for the convention, semantics, and user experience(UX). 👍🏽 
![image](https://user-images.githubusercontent.com/58642949/192073966-f52dbf4b-3a90-4762-8246-666530d1765d.png)